dockerfile

```diff
@@ -7,13 +7,14 @@ FROM python as poetry
 ENV POETRY_HOME=/opt/poetry
 ENV POETRY_VIRTUALENVS_IN_PROJECT=true
 ENV PATH="$POETRY_HOME/bin:$PATH"
+ENV CONFIGURATION_PATH="config/configuration.yml"
```

Comment:
I don't understand why this is needed. Can you explain?

Reply:
That way in the docker compose, we can set this env so that we can just link a volume and define the path to that exact configuration file.